src/server.cpp

```diff
@@ -2554,11 +2569,9 @@ struct SendableMedia
 	std::string path;
 	std::string data;
 
-	SendableMedia(const std::string &name_="", const std::string &path_="",
-	              const std::string &data_=""):
-		name(name_),
-		path(path_),
-		data(data_)
+	SendableMedia(const std::string &name, const std::string &path,
+			std::string &&data):
+		name(name), path(path), data(data)
```

Comment:
`data` is already an [xvalue](https://en.cppreference.com/w/cpp/language/value_category), surely it will call the move constructor of `std::string`, no?

hm, however (when writing the assigment manually):
![grafik](https://user-images.githubusercontent.com/1042418/130135831-605645c4-8b6c-49b8-adde-1ae4bb48b1c2.png)

Reply:
`data` is an lvalue, because it's the name of a variable.
The `&&` means here that `data` binds to rvalues (see [reference declaration](https://en.cppreference.com/w/cpp/language/reference)).